plugins/statistics.koplugin/main.lua

```diff
@@ -1689,6 +1689,19 @@ function ReaderStatistics:getCurrentStat()
         })
     end
 
+    -- Replace estimates for finished/frozen books
+    local estimated_time_left, estimated_finish_date
+    if self.is_doc_not_freezed then
+        estimated_time_left = { _("Estimated reading time left") .. " ⓘ", time_to_read_string, callback = estimated_popup }
+        estimated_finish_date = { _("Estimated finish date") .. " ⓘ", estimates_valid and T(N_("(in 1 day) %2", "(in %1 days) %2", estimate_days_to_read), estimate_days_to_read, estimate_end_of_read_date) or _("N/A"), callback = estimated_popup }
+    else
+        estimated_time_left = { _("Book marked as finished"), _("statistics frozen") }
+        local mark_date = self.ui.doc_settings:readSetting("summary").modified
+        estimated_finish_date = { _("Book marked as finished"), datetime.secondsToDate(datetime.stringToSeconds(mark_date), true) }
```

Comment:
I'd like to have a different wording for the first one that replaces "Estimated reading time left" and indicates that statistics are frozen.
(Having twice the same key, and additionally each having different values, feels quite odd.)
Not sure about what to write. May be `Reading finished       statistics frozen` (or `Reading marked as finished    statistics frozen` but the short one feels more different and better) .

Reply:
I have a mild preference over the ever so slightly more explicit `Book marked as finished` he went with, `Reading finished` doesn't necessarily imply that the book was actually marked as finished, it could be construed as "oh, that happened on its own because I reached the last page" or something ;).

(I mean, you *can* have books auto-flag as finished on EoB, but that's non-default ;p).

Reply:
So, what about the other one I suggested `Reading marked as finished    statistics frozen` ?
(My wish is just to not have twice the same key, and additionally each having different values.)

Reply:
Oh, I'd... possibly approach this another way: don't change the key, just the value? The whole thing is opt-in, so I don't think we need to re-explain why statistics are frozen?

Reply:
Wait, I may be confused; lemme check the full context, as I don't rightly recall how this thing even looks like to begin with ;).

Reply:
Okay, yeah, I would *keep* `estimated_time_left` description as-is, and just change the "value" to mention that stats are frozen.

I do like the change from `Estimated finish date` to `Book marked as finished` date, but this one is technically semantically different for both its description and the value, so it makes sense ;).